bin/v-add-letsencrypt-domain

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
-# info: check letsencrypt domain
+# info: verifies or adds letsencrypt ssl certificate to domain
 # options: USER DOMAIN [ALIASES] [MAIL]
 #
-# example: v-add-letsencrypt-domain admin wonderland.com www.wonderland.com,demo.wonderland.com
-# example: v-add-letsencrypt-domain admin wonderland.com '' yes
+# example: 
+# # USAGE: [ALIASES] full subdomain(s) comma separated with no space i.e. www.domain.tld,sub.domain.tld
```

Comment:
Of we want to explain all options 

It should: ALIAES (Optional) in stead [ALIASES]

Reply:
The intent is to explain non obvious arguments. i.e. we don't need to explain [USER] or [DOMAIN] every time. And [] indicates optional. Perhaps a new doc or doc header in cli.md with generic info like that?